From d6fbb4a796805f8a6a963f745cfb4a704069cfc1 Mon Sep 17 00:00:00 2001
From: Oleksandr Ivantsiv <oivantsiv@nvidia.com>
Date: Tue, 30 Aug 2022 18:40:40 +0300
Subject: [PATCH] Disable bfvcheck service.

- The functionality that the service provides is not needed in SONiC.
We need to have an access only to scripts that the package provides.
---
 debian/mlxbf-scripts.install |  1 -
 debian/postinst              | 17 -----------------
 2 files changed, 18 deletions(-)

diff --git a/debian/mlxbf-scripts.install b/debian/mlxbf-scripts.install
index 10175a1..7aa649d 100644
--- a/debian/mlxbf-scripts.install
+++ b/debian/mlxbf-scripts.install
@@ -1,4 +1,3 @@
 bfacpievt bfbootmgr bfcfg bfcpu-freq bfdracut bffamily bfgrubcheck bfinst usr/bin/
 bfpart bfpxe bfrec bfrshlog bfsbkeys bfvcheck bfver mlx-mkbfb bfhcafw usr/bin/
-bfvcheck.service lib/systemd/system/
 mlx-uefi.quirk usr/share/fwupd/quirks.d/
diff --git a/debian/postinst b/debian/postinst
index f0fd8e4..07fe0c2 100755
--- a/debian/postinst
+++ b/debian/postinst
@@ -1,20 +1,3 @@
 #!/bin/sh -e
-OPTS=""
-
-if [ -n "$D" ]; then
-    OPTS="--root=$D"
-fi
-
-if which deb-systemd-helper >/dev/null 2>/dev/null; then
-	if [ -z "$D" ]; then
-		deb-systemd-helper daemon-reload
-	fi
-
-	deb-systemd-helper "$OPTS" enable bfvcheck.service
-
-	if [ -z "$D" ]; then
-		deb-systemd-helper restart bfvcheck.service
-	fi
-fi
 
 #DEBHELPER#
-- 
2.25.1

